src/main/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionController.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.pip.subscription.management.controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pip.subscription.management.models.Subscription;
+import uk.gov.hmcts.reform.pip.subscription.management.services.SubscriptionService;
+
+import java.util.List;
+
+
+@RestController
+@Api(tags = "Subscription management API")
+@RequestMapping("/subscription")
+public class SubscriptionController {
+
+    @Autowired
+    SubscriptionService subscriptionService;
+
+    @PostMapping(consumes = "application/json", produces = "application/json")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Subscription created"),
+        @ApiResponse(code = 404, message = "Error reaching database")
```

Comment:
A 404 isn't returned in this scenario therefore I think we can remove the 404 @ApiResponse bit